DEVELOPMENT.md

```diff
@@ -19,13 +19,30 @@
 
 The project URL is: https://console.cloud.google.com/home/dashboard?organizationId=854380395992&project=conjur-cloud-launcher-onboard
 
+Before proceeding, make sure that:
+* You are authorized to access to the conjur-cloud-launcher-onboard GCP project.
+* Your current GCP project is set to conjur-cloud-launcher-onboard:
+```
+gcloud config set project conjur-cloud-launcher-onboard
+```
+* Your current GCP cluster is set to the desired cluster in the conjur-cloud-launcher-onboard project:
```

Comment:
conjur-cloud-launcher-onboard should be surrounded by backticks for easier reading